tequilapi/http_middlewares.go

```diff
@@ -7,12 +7,32 @@ type corsHandler struct {
 }
 
 func (wrapper corsHandler) ServeHTTP(resp http.ResponseWriter, req *http.Request) {
-	resp.Header().Set("Access-Control-Allow-Origin", "*")
-	resp.Header().Set("Access-Control-Allow-Methods", "POST, GET, OPTIONS, PUT, DELETE")
+	if isPreflightCorsRequest(req) {
```

Comment:
Idea - would not it be easier to connect middleware for each route, we are registering?

Reply:
It would make sense if we really gonna use CORS for more grained control checking in our api. At the moment - it is simplier to handle all cors (allow all actions) in single place which naturaly is single http handling function.